src/Umbraco.Infrastructure/Runtime/RuntimeEssentialsEventArgs.cs

```diff
@@ -1,23 +1,19 @@
 ﻿using System;
-using Umbraco.Core.Cache;
-using Umbraco.Core.Composing;
+using Umbraco.Core.Builder;
 using Umbraco.Core.Persistence;
 
 namespace Umbraco.Core.Runtime
 {
     public class RuntimeEssentialsEventArgs : EventArgs
     {
-        public RuntimeEssentialsEventArgs(Composition composition, AppCaches appCaches, TypeLoader typeLoader, IUmbracoDatabaseFactory databaseFactory)
```

Comment:
I think this class can be removed for now since I don't think it's actually being used? (see my other comment about RuntimeOptions and Deploy)